Doesn't work on Elvish on Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Evlish's init script doesn't work. I don't know enough about Elvish to tell you why. lol
```
Exception: Compilation error: variable $C:\\Users\\ethan\\AppData\\Local\\Programs\\oh-my-posh\\themes\\hul10.omp.json not found
  [eval 1]:1:98: eval ('C:/Users/ethan/AppData/Local/Programs/oh-my-posh/bin/oh-my-posh.exe' init elvish --config=$'C:\\Users\\ethan\\AppData\\Local\\Programs\\oh-my-posh\\themes\\hul10.omp.json' --print | slurp)
C:\Users\ethan\AppData\Roaming\elvish\rc.elv:1:1: eval (oh-my-posh init elvish)
```

### Theme

%POSH_THEMES_PATH%\hul10.omp.json

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

elvish

### Log output

```Shell
Version: 14.30.0

Shell: elvish

Segments:

ConsoleTitle(true)                         -   1 ms - elvish in <Windows>
text(true)                                 -   0 ms - ┌
os(true)                                   -   0 ms -  ⠀
shell(true)                                -   0 ms -   elvish⠀
session(true)                              -   0 ms -   Ethan-PCethan⠀
git(false)                                 -  13 ms -
executiontime(true)                        -   0 ms - 0ms
time(true)                                 -   0 ms -  · 21/04/23 12:55
path(true)                                 -   1 ms - └[ C: » Windows ]
exit(true)                                 -   0 ms - ❯

Run duration: 34.3568ms

Cache path: C:\Users\ethan\AppData\Local\oh-my-posh

Config path: C:\Users\ethan\AppData\Local\Programs\oh-my-posh\themes\hul10.omp.json

Logs:

[DEBUG] 12:55:29.849 shell.go:Getenv:398 → NO DATA
[TRACE] 12:55:29.849 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 12:55:29.849 shell.go:Getenv:398 → C:\Users\ethan\AppData\Local
[TRACE] 12:55:29.849 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 12:55:29.849 shell_windows.go:CachePath() - 0s
[DEBUG] 12:55:29.851 shell.go:Getenv:398 → C:\Users\ethan\AppData\Local\Programs\oh-my-posh\themes\hul10.omp.json
[TRACE] 12:55:29.851 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 12:55:29.863 shell.go:Shell:645 → process name: elvish.exe
[TRACE] 12:55:29.863 shell.go:Shell() - 12.001ms
[TRACE] 12:55:29.863 shell.go:resolveConfigPath() - 12.001ms
[TRACE] 12:55:29.863 shell.go:Init() - 14.033ms
[TRACE] 12:55:29.863 shell.go:Flags() - 0s
[TRACE] 12:55:29.864 config.go:loadConfig() - 1.1614ms
[TRACE] 12:55:29.864 shell.go:Flags() - 0s
[DEBUG] 12:55:29.864 shell.go:Getenv:398 → NO DATA
[TRACE] 12:55:29.864 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 12:55:29.864 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.864 shell.go:GOOS() - 0s
[TRACE] 12:55:29.865 shell.go:GOOS() - 0s
[TRACE] 12:55:29.865 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.865 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40C2F69
[TRACE] 12:55:29.865 shell.go:Shell() - 0s
[DEBUG] 12:55:29.865 shell.go:Getenv:398 → NO DATA
[TRACE] 12:55:29.865 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 12:55:29.865 engine.go:PrintDebug:266 → Segment: Title
[TRACE] 12:55:29.865 shell_windows.go:Root() - 0s
[TRACE] 12:55:29.865 shell.go:Shell() - 0s
[TRACE] 12:55:29.865 shell.go:ErrorCode() - 0s
[TRACE] 12:55:29.865 shell_windows.go:IsWsl() - 0s
[TRACE] 12:55:29.865 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.866 shell.go:Pwd:428 → C:\Windows
[TRACE] 12:55:29.866 shell.go:Pwd() - 499.4µs
[DEBUG] 12:55:29.866 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.866 shell.go:User:549 → ethan
[TRACE] 12:55:29.866 shell.go:User() - 0s
[DEBUG] 12:55:29.866 shell.go:Host:561 → Ethan-PC
[TRACE] 12:55:29.866 shell.go:Host() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.866 shell.go:Getenv:398 → 1
[TRACE] 12:55:29.866 shell.go:Getenv(SHLVL) - 0s
[TRACE] 12:55:29.866 shell.go:TemplateCache() - 499.4µs
[TRACE] 12:55:29.866 shell.go:Flags() - 0s
[DEBUG] 12:55:29.866 block.go:Debug:230 → Segment: text
[TRACE] 12:55:29.866 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.866 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.866 shell.go:Shell() - 0s
[DEBUG] 12:55:29.866 block.go:Debug:230 → Segment: os
[TRACE] 12:55:29.866 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.866 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.866 properties.go:GetString:28 → 
[TRACE] 12:55:29.866 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.866 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.866 shell.go:Shell() - 0s
[DEBUG] 12:55:29.866 block.go:Debug:230 → Segment: shell
[TRACE] 12:55:29.866 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.866 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.866 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.866 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 12:55:29.866 shell.go:Shell() - 0s
[TRACE] 12:55:29.866 shell.go:Flags() - 0s
[TRACE] 12:55:29.866 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.867 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.867 shell.go:Shell() - 0s
[DEBUG] 12:55:29.867 block.go:Debug:230 → Segment: session
[TRACE] 12:55:29.867 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.867 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.867 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.867 shell.go:Getenv:398 → NO DATA
[TRACE] 12:55:29.867 shell.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 12:55:29.867 shell.go:Getenv:398 → NO DATA
[TRACE] 12:55:29.867 shell.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 12:55:29.867 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.867 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.867 shell.go:Shell() - 0s
[DEBUG] 12:55:29.867 block.go:Debug:230 → Segment: git
[TRACE] 12:55:29.867 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.867 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.867 shell.go:GOOS() - 0s
[TRACE] 12:55:29.867 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.881 shell.go:CommandPath:600 → C:\Users\ethan\scoop\shims\git.exe
[TRACE] 12:55:29.881 shell.go:CommandPath(git.exe) - 13.7969ms
[TRACE] 12:55:29.881 shell.go:HasCommand(git.exe) - 13.7969ms
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[ERROR] 12:55:29.881 shell.go:HasParentFilePath:733 → CreateFile .git: The system cannot find the file specified.
[TRACE] 12:55:29.881 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 12:55:29.881 properties.go:GetBool:22 → fetch_bare_info: false
[TRACE] 12:55:29.881 shell.go:Flags() - 0s
[DEBUG] 12:55:29.881 block.go:Debug:230 → Segment: executiontime
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.881 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.881 properties.go:GetBool:22 → always_enabled: false
[TRACE] 12:55:29.881 shell.go:ExecutionTime() - 0s
[DEBUG] 12:55:29.881 properties.go:GetFloat64:34 → threshold: 0.000000
[DEBUG] 12:55:29.881 properties.go:GetString:28 → austin
[TRACE] 12:55:29.881 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.881 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.881 shell.go:Shell() - 0s
[DEBUG] 12:55:29.881 block.go:Debug:230 → Segment: time
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.881 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.881 properties.go:GetString:28 → 02/01/06 15:04
[TRACE] 12:55:29.881 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.881 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.881 shell.go:Shell() - 0s
[TRACE] 12:55:29.881 shell.go:Flags() - 0s
[DEBUG] 12:55:29.881 block.go:Debug:230 → Segment: path
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.881 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[TRACE] 12:55:29.881 shell.go:Shell() - 0s
[TRACE] 12:55:29.881 shell.go:Shell() - 0s
[DEBUG] 12:55:29.881 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 12:55:29.881 properties.go:GetString:28 → 
[DEBUG] 12:55:29.881 properties.go:GetString:28 → 
[DEBUG] 12:55:29.881 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[DEBUG] 12:55:29.881 properties.go:GetString:28 → 
[DEBUG] 12:55:29.881 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:GOOS() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[DEBUG] 12:55:29.881 properties.go:GetString:28 → agnoster
[DEBUG] 12:55:29.881 properties.go:GetString:28 → 
[DEBUG] 12:55:29.881 properties.go:GetString:28 → <#93d0ff> » </>
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:PathSeparator() - 0s
[TRACE] 12:55:29.881 shell.go:Pwd() - 0s
[TRACE] 12:55:29.881 shell_windows.go:IsWsl() - 0s
[TRACE] 12:55:29.881 shell.go:StackCount() - 0s
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:311 ↓
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:311 → GENERIC_ALL
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-545
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:311 ↓
    SYNCHRONIZE
    READ_CONTROL
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-545
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:311 ↓
    GENERIC_READ
    GENERIC_EXECUTE
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 12:55:29.883 win32_windows.go:isWriteable:317 → no write access
[TRACE] 12:55:29.883 shell_windows.go:DirIsWritable() - 1.5331ms
[TRACE] 12:55:29.883 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.883 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.883 shell.go:Shell() - 0s
[TRACE] 12:55:29.883 shell.go:Flags() - 0s
[DEBUG] 12:55:29.883 block.go:Debug:230 → Segment: exit
[TRACE] 12:55:29.883 shell.go:Pwd() - 0s
[DEBUG] 12:55:29.883 shell_windows.go:56 → C:\Users\ethan
[TRACE] 12:55:29.883 shell.go:GOOS() - 0s
[TRACE] 12:55:29.883 shell.go:ErrorCode() - 0s
[DEBUG] 12:55:29.883 properties.go:GetBool:22 → always_enabled: true
[TRACE] 12:55:29.883 shell.go:TemplateCache() - 0s
[TRACE] 12:55:29.883 shell.go:Shell() - 0s
[TRACE] 12:55:29.883 shell.go:TemplateCache() - 0s
[DEBUG] 12:55:29.883 shell.go:Getenv:398 → C:\Users\ethan\AppData\Local
[TRACE] 12:55:29.883 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 12:55:29.883 shell_windows.go:CachePath() - 0s
[TRACE] 12:55:29.883 shell.go:Flags() - 0s
```

--- a/src/shell/init.go
+++ b/src/shell/init.go
@@ -180,11 +180,12 @@ func Init(env platform.Environment) string {
 		case PWSH, PWSH5:
 			command = "(@(& %s init %s --config=%s --print%s) -join \"`n\") | Invoke-Expression"
 			config = quotePwshStr(env.Flags().Config)
+			executable = quotePwshStr(executable)
 		case ELVISH:
 			command = "eval (%s init %s --config=%s --print%s | slurp)"
-			config = quotePosixStr(env.Flags().Config)
+			config = env.Flags().Config
 		}
-		return fmt.Sprintf(command, quotePwshStr(executable), shell, config, additionalParams)
+		return fmt.Sprintf(command, executable, shell, config, additionalParams)
 	case ZSH, BASH, FISH, CMD, TCSH, XONSH:
 		return PrintInit(env)
 	case NU:
@@ -246,12 +247,8 @@ func PrintInit(env platform.Environment) string {
 		configFile = quotePosixStr(configFile)
 		script = tcshInit
 	case ELVISH:
-		executable = quotePosixStr(executable)
-		configFile = quotePosixStr(configFile)
 		script = elvishInit
 	case XONSH:
-		executable = quotePosixStr(executable)
-		configFile = quotePosixStr(configFile)
 		script = xonshInit
 	default:
 		return fmt.Sprintf("echo \"No initialization script available for %s\"", shell)

